File: Daily_Codes/oct05/Program03.java
class Parent {

	Parent(){
		System.out.println(this.hashCode());
		System.out.println("Parent Constructor");
	}
}

class Child extends Parent {

	Child(){
		System.out.println(this.hashCode());
		System.out.println(super.hashCode());
		System.out.println("Child Constructor");
	}

}
public class Program03{
	public static void main(String[] args){
		
		Child obj = new Child(); // Child();

	}
}

File: Daily_Codes/oct05/Program03.dart
class Parent {
	
	Parent(){			// Parent(Parent this)
		print(this.hashCode);
	}
}

class Child extends Parent {

	Child(){			// Child(Child this)
		print(super.hashCode);	// super => (Parent)this
	}
}

void main(){
	
	Child obj = new Child(); 	// Child(obj)	
	print(obj.hashCode);
}
